commit 63b6ced63a21751201618ae98e6a25f083fe749a (refs/changes/36/2636/3)
Author: Angela Fong <angela.fong@joyent.com>
Date:   2017-09-22T09:17:02-07:00 (2 years, 1 month ago)
    
    ADMINUI-2373 Want to Show Disk Layout Type in Server Details
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/www/js/tpl/server.hbs b/www/js/tpl/server.hbs
index 54575362..b8b1683e 100644
--- a/www/js/tpl/server.hbs
+++ b/www/js/tpl/server.hbs
@@ -168,6 +168,10 @@
           <th>Total Disk Space</th>
           <td>{{total_quota}} GB</td>
         </tr>
+        <tr>
+          <th>Disk Layout</th>
+          <td>{{disk_layout}}</td>
+        </tr>
       </tbody>
     </table>
   </div>
diff --git a/www/js/views/server.js b/www/js/views/server.js
index 6e3d5d97..07c9d262 100644
--- a/www/js/views/server.js
+++ b/www/js/views/server.js
@@ -172,6 +172,9 @@ var ServerView = Backbone.Marionette.Layout.extend({
         },
         total_memory: function () {
             return this.sysinfo['MiB of Memory'];
+        },
+        disk_layout: function () {
+            return this.sysinfo['Zpool Profile'];
         }
     },
 
@@ -183,7 +186,6 @@ var ServerView = Backbone.Marionette.Layout.extend({
             data.total_quota += v['Size in GB'];
         });
 
-
         data.disks = _.map(data.sysinfo['Disks'], function (v, k) {
             return {
                 name: k,
